aws_cloudformation_policies/aws_cloudformation_stack_drifted.py

```diff
@@ -1,2 +1,20 @@
+# CloudFormation stacks tagged with "STACK=(name)" will be marked as passing.
+IGNORE_STACK_TAGS = {
+    'panther-bootstrap-gateway',
+    'panther-cloud-security',
+    'panther-core',
+    'panther-log-analysis'
+}
+
+
 def policy(resource):
-    return resource['DriftInformation']['StackDriftStatus'] != "DRIFTED"
+    if resource['DriftInformation']['StackDriftStatus'] != "DRIFTED":
+        return True
+
+    # Some of Panther's own stacks contain Lambda functions which will always show as "drifted."
+    # Panther stacks have a fixed "Stack" tag, even though the real stack name is dynamic.
```

Comment:
I do this ignoring in the Python rather than `.yml` metadata because we don't know what the stack arn will be - the user has full control over the stack name. I have to rely on the tags instead, which we set explicitly